data/lib/core/player.lua

```diff
@@ -384,6 +384,80 @@ function Player.setSpecialContainersAvailable(self, available)
 	return true
 end
 
+function Player.getMaxTrackedQuests(self)
+	return configManager.getNumber(self:isPremium() and configKeys.QUEST_TRACKER_PREMIUM_LIMIT or configKeys.QUEST_TRACKER_FREE_LIMIT)
+end
+
+function Player.sendQuestLog(self)
+	local msg = NetworkMessage()
+	msg:addByte(0xF0)
+	local quests = Quest.getQuests(self)
+	msg:addU16(#quests)
+
+	for _, quest in pairs(quests) do
+		msg:addU16(quest.id)
+		msg:addString(quest.name)
+		msg:addByte(quest:isCompleted(self))
+	end
+
+	msg:sendToPlayer(self)
+	msg:delete()
+	return true
+end
+
+function Player.sendQuestLine(self, quest)
+	local msg = NetworkMessage()
+	msg:addByte(0xF1)
+	msg:addU16(quest.id)
+	local missions = quest:getMissions(self)
+	msg:addByte(#missions)
+
+	for _, mission in pairs(missions) do
+		msg:addU16(mission.id)
+		msg:addString(mission:getName(self))
+		msg:addString(mission:getDescription(self))
+	end
+
+	msg:sendToPlayer(self)
+	msg:delete()
+	return true
+end
+
+function Player.sendQuestTracker(self, missionsId)
+	local msg = NetworkMessage()
+	msg:addByte(0xD0)
+	msg:addByte(1)
+
+	local trackedQuests, trackeds = TrackedQuests(self, missionsId)
```

Comment:
`TrackedQuests` is a bad name, makes it look like you are constructing a new value, but it's just a function. `getTrackedQuests` maybe? And you don't need to return `trackeds` just use `#trackedQuests`